plugins/woocommerce/tests/performance/requests/shopper/my-account-orders.js

```diff
@@ -29,6 +29,7 @@ import { checkResponse } from '../../utils.js';
 
 export function myAccountOrders() {
 	let my_account_order_id;
+	let response;
```

Comment:
Why are you re-using a global response variable? I don't see how this helps in fixing the test.

Reply:
I reverted the change, thanks for spotting this. It worked in both ways, but at some point, I thought this blocked the test.